BunqSdk.Tests/Model/Generated/Endpoint/TabUsageSingleTest.cs

```diff
@@ -16,12 +16,12 @@ public class TabUsageSingleTest : BunqSdkTestBase
         /// <summary>
         /// Config values
         /// </summary>
-        private const string TabDescription = "Pay the tab for Java test please.";
-        private const string StatusOpen = "OPEN";
-        private const string Amount = "42.00";
-        private const string Currency = "EUR";
-        private const string TabItemDescription = "Super expensive java tea";
-        private const string StatusWaiting = "WAITING_FOR_PAYMENT";
+        private const string TabUsageSingleDescription = "Pay the tab for Java test please.";
+        private const string TabUsageSingleStatusOpen = "OPEN";
+        private const string TabAmountEur = "42.00";
```

Comment:
Please be consistent 😝 `TabAmountEur` -> `TabUsageSingleAmountEur`

Reply:
The constant is also used along with the TabItemShop entity. Changing it for consistency!